recipes/r-forestfloor/build.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+if [[ $target_platform =~ linux.* ]] || [[ $target_platform == win-32 ]] || [[ $target_platform == win-64 ]] || [[ $target_platform == osx-64 ]]; then
+  export DISABLE_AUTOBREW=1
+  X11_CONFIGURE_ARGS="--x-includes=${PREFIX}/include --x-libraries=${BUILD_PREFIX}/x86_64-conda_cos6-linux-gnu/sysroot/usr/lib64"
+  export LD_LIBRARY_PATH=${BUILD_PREFIX}/x86_64-conda_cos6-linux-gnu/sysroot/usr/lib64
```

Comment:
Why is this necessary?

Reply:
I stole it from [r-rgl](https://github.com/conda-forge/r-rgl-feedstock/blob/master/recipe/build.sh). Without building fails with missing `libGL.so.1` error. If you know a better solution I'm happy see.

Reply:
@isuruf I shamelessly stole it from [r-rgl](https://github.com/conda-forge/r-rgl-feedstock/blob/master/recipe/build.sh). Without building fails with missing libGL.so.1 error. If you know a better solution I'm happy see.

Reply:
Do you have a log?

Reply:
@isuruf here the relevant part
```
...
** package ‘forestFloor’ successfully unpacked and MD5 sums checked
** libs
installing to /opt/conda/conda-bld/r-forestfloor_1563684818758/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/R/library/forestFloor/libs
** R
** inst
** byte-compile and prepare package for lazy loading
Error in dyn.load(file, DLLpath = DLLpath, ...) : 
  unable to load shared object '/opt/conda/conda-bld/r-forestfloor_1563684818758/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/R/library/rgl/libs/rgl.so':
  libGL.so.1: cannot open shared object file: No such file or directory
Error : .onLoad failed in loadNamespace() for 'rgl', details:
  call: NULL
  error:        Loading rgl's DLL failed. 
ERROR: lazy loading failed for package ‘forestFloor’
* removing ‘/opt/conda/conda-bld/r-forestfloor_1563684818758/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/R/library/forestFloor’
-GCC_RANLIB=$PREFIX/bin/x86_64-conda_cos6-linux-gnu-gcc-ranlib
INFO: activate-gxx_linux-64.sh made the following environmental changes:
+CXX=$BUILD_PREFIX/bin/x86_64-conda_cos6-linux-gnu-c++
-CXX=$PREFIX/bin/x86_64-conda_cos6-linux-gnu-c++
+GXX=$BUILD_PREFIX/bin/x86_64-conda_cos6-linux-gnu-g++
-GXX=$PREFIX/bin/x86_64-conda_cos6-linux-gnu-g++
x86_64-conda_cos6-linux-gnu-c++  -I"$PREFIX/lib/R/include" -DNDEBUG  -I"$PREFIX/lib/R/library/Rcpp/include" -DNDEBUG -D_FORTIFY_SOURCE=2 -O2  -I$PREFIX/include -Wl,-rpath-link,$PREFIX/lib   -fpic  -fvisibility-inlines-hidden  -fmessage-length=0 -march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O2 -ffunction-sections -pipe -I$PREFIX/include -fdebug-prefix-map=/home/conda/feedstock_root/build_artifacts/r-base_1562026475051/work=/usr/local/src/conda/r-base-3.5.1 -fdebug-prefix-map=$PREFIX=/usr/local/src/conda-prefix  -c RcppExports.cpp -o RcppExports.o
x86_64-conda_cos6-linux-gnu-c++  -I"$PREFIX/lib/R/include" -DNDEBUG  -I"$PREFIX/lib/R/library/Rcpp/include" -DNDEBUG -D_FORTIFY_SOURCE=2 -O2  -I$PREFIX/include -Wl,-rpath-link,$PREFIX/lib   -fpic  -fvisibility-inlines-hidden  -fmessage-length=0 -march=nocona -mtune=haswell -ftree-vectorize -fPIC -fstack-protector-strong -fno-plt -O2 -ffunction-sections -pipe -I$PREFIX/include -fdebug-prefix-map=/home/conda/feedstock_root/build_artifacts/r-base_1562026475051/work=/usr/local/src/conda/r-base-3.5.1 -fdebug-prefix-map=$PREFIX=/usr/local/src/conda-prefix  -c testcpp_rec6.cpp -o testcpp_rec6.o
x86_64-conda_cos6-linux-gnu-c++ -shared -L$PREFIX/lib/R/lib -Wl,-O2 -Wl,--sort-common -Wl,--as-needed -Wl,-z,relro -Wl,-z,now -Wl,--disable-new-dtags -Wl,--gc-sections -Wl,-rpath,$PREFIX/lib -Wl,-rpath-link,$PREFIX/lib -L$PREFIX/lib -Wl,-rpath-link,$PREFIX/lib -o forestFloor.so RcppExports.o testcpp_rec6.o -L$PREFIX/lib/R/lib -lR
Traceback (most recent call last):
  File "/opt/conda/bin/conda-build", line 11, in <module>
    sys.exit(main())
  File "/opt/conda/lib/python3.7/site-packages/conda_build/cli/main_build.py", line 445, in main
    execute(sys.argv[1:])
  File "/opt/conda/lib/python3.7/site-packages/conda_build/cli/main_build.py", line 436, in execute
    verify=args.verify, variants=args.variants)
  File "/opt/conda/lib/python3.7/site-packages/conda_build/api.py", line 209, in build
    notest=notest, need_source_download=need_source_download, variants=variants)
  File "/opt/conda/lib/python3.7/site-packages/conda_build/build.py", line 2324, in build_tree
    notest=notest,
  File "/opt/conda/lib/python3.7/site-packages/conda_build/build.py", line 1472, in build
    cwd=src_dir, stats=build_stats)
```

Reply:
You need a `yum_requirements.txt` file like https://github.com/conda-forge/fltk-feedstock/blob/fcb778d2fd11859214323ac4bf0361354fef9451/recipe/yum_requirements.txt

Reply:
@isuruf Thx! I've adapted accordingly, removing `export LD_LIBRARY_PATH= ...` and added a `yum_requirements.txt`